src/main/steps/common/form/form.njk

```diff
@@ -23,7 +23,7 @@
             value: true,
             preventDoubleClick: true,
             attributes: {
-              id: 'main-form-submit'
```

Comment:
Is the id used anywhere in the code? Is the change tested and working without breaking any functionality.

Reply:
Not used anywhere else in application, no breaking functionality observed during testing

Reply:
Reverted this change, its already under conditional, based on accesscodecheck - no need to change the id for this field